docs/conditions.md

```diff
@@ -368,6 +368,58 @@ The following parameters are expected:
 
 ---
 
+### 66 `SEND_MESSAGE` {#send-message}
+
+:::info
```

Comment:
it looks like I got this syntax wrong

Reply:
GH rich diff does not properly render the info boxes. The live deployment of the pr is properly rendering those boxes